samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -66,6 +66,32 @@ public DocumentTranslationClient(string endpoint, AzureKeyCredential credential,
         /// <param name="context"> The request context, which can override default behaviors of the client pipeline on a per-call basis. </param>
         /// <exception cref="RequestFailedException"> Service returned a non-success status code. </exception>
         /// <returns> The response returned from the service. Details of the response body schema are in the Remarks section below. </returns>
+        /// <example>
+        /// This sample shows how to call GetDocumentStatusAsync with required parameters and parse the result.
+        /// <code><![CDATA[
+        /// var credential = new AzureKeyCredential("<key>");
+        /// var client = new DocumentTranslationClient("<https://my-account-name.azure.com>", credential);
+        /// 
+        /// Response response = await client.GetDocumentStatusAsync(Guid.NewGuid(), Guid.NewGuid());
+        /// 
+        /// JsonElement result = JsonDocument.Parse(response.ContentStream).RootElement;
+        /// Console.WriteLine(result.GetProperty("path").ToString());
+        /// Console.WriteLine(result.GetProperty("sourcePath").ToString());
+        /// Console.WriteLine(result.GetProperty("createdDateTimeUtc").ToString());
+        /// Console.WriteLine(result.GetProperty("lastActionDateTimeUtc").ToString());
+        /// Console.WriteLine(result.GetProperty("status").ToString());
+        /// Console.WriteLine(result.GetProperty("to").ToString());
+        /// Console.WriteLine(result.GetProperty("error").GetProperty("code").ToString());
```

Comment:
For the call of GetProperty, will the result be null which may cause unexpected behaviour?

Reply:
I think here we have to make assumption that is data returned from backend. Otherwise we cannot show how to parse the response, or the defensive programming codes will be too long.

Reply:
Can we use Null-conditional operators here?

Reply:
The return value is `JsonElement`, not `JsonElement?`. So I think we'd better keep it simple here.